src/main/resources/mybatis/timeSeries.xml

```diff
@@ -9,6 +9,20 @@
         on gw.monitoring_location_id = ml.monitoring_location_id
     </sql>
 
+    <sql id="bestTimeSeriesQuery">
+       select monitoring_location_identifier, time_series_unique_id best_ts_id, parameter_cd, statistic_cd
+       from (select *, RANK() OVER (ORDER BY ranking DESC) AS rk
+             from nwis.best_time_series
+             where monitoring_location_identifier = #{featureId,jdbcType=VARCHAR}
+                   and ranking > 0) as t
+       where rk = 1
+    </sql>
+
+    <sql id="bestTimeSeriesJoin">
+       left join lateral (<include refid="bestTimeSeriesQuery"/>) best_time_series
+       on ml.monitoring_location_identifier = best_time_series.monitoring_location_identifier
+    </sql>
+
```

Comment:
I haven't used a `lateral` before, but from some quick googling, I think it is not required here, maybe?  Your sub-select doesn't reference anything else in the query.

Reply:
Good catch. Left over for some testing. Yup, not needed.